tests/044_bad_resource.ts

```diff
@@ -0,0 +1,7 @@
+async function main(): Promise<void> {
```

Comment:
Test that is failing on master:
```
async function main(): Promise<void> {
  const file = await Deno.open("Cargo.toml", "a");
  file.write(new Uint8Array([0, 1, 2]));
  file.close();
}

main();

```

Unfortunately minimal dispatch still swallows errors and it can't be used right now (because it'll throw generic write error instead of displaying actual error)

Reply:
Yeah, minimal_dispatch doesn't have error handling... I need to get to that.